api/src/test/java/org/openmrs/module/idgen/service/IdentifierSourceServiceTest.java

```diff
@@ -110,6 +110,19 @@ public void getIdentifierSource_shouldReturnASavedIdentifierPool() throws Except
 		Assert.assertEquals(5, idpool.getAvailableIdentifiers().size());
 		Assert.assertEquals(idpool.getUsedIdentifiers().size(), 0);
 	}
+	
+	/**
+	 * @see {@link IdentifierSourceService#retireIdentifierSource(IdentifierSource, String)}
+	 * 
+	 */
+	@Test
+	@Verifies(value = "should retire an IdentifierSource from the system", method = "retireIdentifierSource(IdentifierSource, String)")
+	public void retireIdentifierSource_shouldRetireAnIdentifierSourceFromTheSystem() throws Exception {
+		IdentifierSource s = iss.getIdentifierSource(1);

```

Comment:
variable name is not good. Can you rename it to identifierSource?

Reply:
fixed